intellij/src/saros/core/ui/eventhandler/NegotiationHandler.java

```diff
@@ -136,33 +136,41 @@ protected IStatus run(IProgressMonitor monitor) {
           case CANCEL:
             return Status.CANCEL_STATUS;
           case ERROR:
+            NotificationPanel.showError(
+                MessageFormat.format(
+                    Messages.NegotiationHandler_session_local_error_message,
+                    negotiation.getErrorMessage()),
+                Messages.NegotiationHandler_session_local_error_title);
+
             return new Status(
                 IStatus.ERROR, SarosComponent.PLUGIN_ID, negotiation.getErrorMessage());
           case OK:
             break;
           case REMOTE_CANCEL:
             NotificationPanel.showInformation(
-                MessageFormat.format(Messages.NegotiationHandler_canceled_invitation_text, peer),
-                Messages.NegotiationHandler_canceled_invitation);
+                MessageFormat.format(
```

Comment:
Duplicated code. Maybe you can declare: String title, message at the top of the method an then assign the values. This is easier to read because you generate the message e.g 2 times here in this section.

Reply:
Done.